bundle.Dockerfile

```diff
@@ -5,7 +5,8 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=sailoperator
-LABEL operators.operatorframework.io.bundle.channels.v1="0.2"
+LABEL operators.operatorframework.io.bundle.channels.v1="0.2-dev"
+LABEL operators.operatorframework.io.bundle.channel.default.v1=0.2-dev
```

Comment:
Is this needed when we have just one channel? I believe this is only useful when having multiple channels in `operators.operatorframework.io.bundle.channel.default.v1`

Reply:
good point. removed